tests/dgstrf.rs

```diff
@@ -0,0 +1,214 @@
+extern crate superlu_sys as raw;
+extern crate libc;
+
+// https://github.com/copies/superlu/blob/master/EXAMPLE/superlu.c
+#[allow(non_snake_case)]
+#[test]
+fn test_dgstrf_dgstrs() {
+    use std::mem::MaybeUninit;
+    use std::slice::from_raw_parts_mut;
+    use libc::c_int;
+
+    use raw::*;
+    use raw::Dtype_t::*;
+    use raw::Mtype_t::*;
+    use raw::Stype_t::*;
+    use raw::colperm_t::*;
+    use raw::trans_t::*;
+    use raw::fact_t::*;
```

Comment:
Perhaps `rustfmt`?

Reply:
note the original test is not an identity transform under rustfmt either  (it has colperm_t after the others)